src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -11335,6 +11335,1929 @@ public override BoundNode VisitExpressionWithNullability(BoundExpressionWithNull
         }
     }
 
+    internal sealed partial class NullabilityRewriter : BoundTreeRewriter
+    {
+        private readonly ImmutableDictionary<BoundExpression, TypeSymbolWithAnnotations> _topLevelNullabilities;
+
+        public NullabilityRewriter(ImmutableDictionary<BoundExpression, TypeSymbolWithAnnotations> topLevelNullabilities)
+        {
+            _topLevelNullabilities = topLevelNullabilities;
+        }
+
+        public override BoundNode VisitDeconstructValuePlaceholder(BoundDeconstructValuePlaceholder node)
+        {
+            if (!_topLevelNullabilities.ContainsKey(node))
+            {
+                return node;
+            }
+
+            TypeSymbol type = _topLevelNullabilities[node].TypeSymbol;
```

Comment:
Please use `_topLevelNullabilities.TryGetValue(node, out type)` to avoid the extract lookup.